### PR TITLE
chore: add explict parity acknowledgement on cert & verifier files, clarify codebase copyright is assigned to n0, inc.

### PR DESCRIPTION
**Pull request by b5**
Two files in our codebase are largely pulled from `libp2p-tls`, and while both projects are MIT licensed, we should clarify and acknowledge that these two files originated from an MIT license Copy-written by Parity Technologies (UK), Ltd. 

Shoutout to Parity who both originated this code and contributed it to the libp2p project, and the libp2p, team that has extended this code. Also thank you to @divagant-martian for realizing this was missing.

At the same time, I've also added "Copyright N0, INC", to clarify the entity this code has actually been assigned to

### CODE REVIEW

File: LICENSE-MIT [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2023
+Copyright 2024 N0, INC.
```
Comment (divagant-martian) [inline LICENSE-MIT:1]: why 2024?

Comment (b5) [reply LICENSE-MIT:1]: LOL my bad, fixing.

File: iroh-net/src/tls/certificate.rs [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+// DEALINGS IN THE SOFTWARE.
```
[the remark this answers is not in the document]
Comment (divagant-martian) [reply iroh-net/src/tls/certificate.rs:19]: I like this format, looks like a nice middle ground

**Review by divagant-martian — APPROVED** [review]
don't we all love github loopholes? lgtm

**Pull request merged**